src/utils.js

```diff
@@ -47,19 +51,47 @@ exp.sorting = (a, b, format) => {
 }
 
 /**
- * Get the BIRTHDAY_CHANNEL_MESSAGE list and switches the status to the next message.
+ * Get the message for specified event depended on the specified condition.
  *
+ * @param {String} event - Name of the event.
+ * @param {String} condition - Condition which specified how to get message.
  * @returns {String}
  */
-exp.getBirthdayChannelMessage = () => {
-  const result = exp.BIRTHDAY_CHANNEL_MESSAGE[BIRTHDAY_CHANNEL_MESSAGE_INDEX]
+exp.getMessage = (event, condition) => {
+  let fileName
+  let index
+
+  if (event === exp.FWD_EVENT_TYPE) {
+    fileName = '/anniversary.txt'
+    index = FWD_MESSAGE_INDEX
+  }
 
-  BIRTHDAY_CHANNEL_MESSAGE_INDEX++
-  if (BIRTHDAY_CHANNEL_MESSAGE_INDEX >= exp.BIRTHDAY_CHANNEL_MESSAGE.length) {
-    BIRTHDAY_CHANNEL_MESSAGE_INDEX = 0
+  if (event === exp.BDAY_EVENT_TYPE) {
+    fileName = '/quotes.txt'
+    index = BIRTHDAY_CHANNEL_MESSAGE_INDEX
   }
 
-  return result
+  const quotesPath = path.join(__dirname, fileName)
+  let quotes = fs.readFileSync(quotesPath, 'utf8').toString().split('\n')
```

Comment:
`fs.readFileSync` is already returns String type. There is no need to make it string